components/raftstore/src/store/worker/pd.rs

```diff
@@ -200,6 +201,15 @@ pub struct PeerStat {
     pub approximate_size: u64,
 }
 
+impl PeerStat {
+    pub fn new(peer_id: u64) -> Self {
+        PeerStat {
+            peer_id: Some(peer_id),
```

Comment:
When can it be `None`?

Reply:
When its initialization is triggered by read stats reporting:
https://github.com/tikv/tikv/blob/8962514cc5be60cc7903e40a8925a95984e9a58a/components/raftstore/src/store/worker/pd.rs#L920-L925

Reply:
Instead of introducing a new field, how about resetting it to zero when stats from heartbeat is smaller than last stats?

Reply:
@BusyJay I think the metrics overflow is caused by a recently introduced bug, but I'm not familiar with raftstore to understand the severity of it. I'm also okay with the simpler version if you think this won't evolve into a bigger problem.